code/modules/events/electrical_storm.dm

```diff
@@ -1,8 +1,7 @@
 /datum/round_event_control/electrical_storm
 	name = "Electrical Storm"
 	typepath = /datum/round_event/electrical_storm
-	earliest_start = 6000
-	weight = 40
+	average_time = 5 //Minor event, no permanent damage

```

Comment:
It's annoying, but it's not too dangerous or difficult to deal with, so it's appropriate for the early game.  Bear in mind that with this system, unlike with the old one, having it early doesn't mean that you will have it often.  I think the main reason electrical storms were so annoying was that you'd often have several in one round.